pkg/agent/entrypoint.go

```diff
@@ -31,8 +31,10 @@ func (r MainRunner) Run(ctx context.Context) error {
 	klog.Infof("buildInfo.GitInfo:   %s", buildInfo.GitInfo)
 	klog.Infof("buildInfo.GoVersion: %s", buildInfo.GoVersion)
 
+	watchMetrics := watch.NewMetrics("autoscaling_agent_watchers_")
```

Comment:
trailing underscore?

Reply:
Yeah... Something's gotta have the leading/trailing underscore. Although on second thought I'd prefer that to be internal to the `watch` package. Changed.